pkg/mcs/resourcemanager/server/keyspace_manager_test.go

```diff
@@ -255,11 +255,6 @@ func TestGetResourceGroupList(t *testing.T) {
 	re.Len(groups, 4)
 	groups = krgm.getResourceGroupList(false, false)
 	re.Len(groups, 3)
-
-	names := krgm.getResourceGroupNames(true)
-	re.Len(names, 4)
-	names = krgm.getResourceGroupNames(false)
-	re.Len(names, 3)
```

Comment:
why remove it?

Reply:
Because `getResourceGroupNames` method is removed.